src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+
+    <session-factory>
+
+        <!-- JDBC Database connection settings -->
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://134.2.24.116:3306/qbic_usermanagement_db?useSSL=false&amp;serverTimezone=UTC</property>
+        <property name="connection.username">qbicuser</property>
+        <property name="connection.password">*G@#588Naq1yrkSc1d2RL2Pt5a%wgr8S</property>
```

Comment:
remind to the future me: use f* env vars!

Reply:
will do :+1: